data/build.gradle

```diff
@@ -1,27 +1,18 @@
 plugins {
-    id 'com.android.application'
+    id 'com.android.library'
     id 'org.jetbrains.kotlin.android'
 }
 
 android {
     compileSdk 31
 
     defaultConfig {
-        applicationId "com.example.data"
         minSdk 21
         targetSdk 31
-        versionCode 1
-        versionName "1.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-    }
```

Comment:
library modules cannot declare `applicationId`, `versionCode` and `versionName` (since that's declared in main "presentation" module)
It also doesn't have to declare `buildTypes` if it's not adding anything new (`minifyEnabled` and `proguardFiles` are already defined like this in `presentation/build.gradle`)